pkg/controllers/dynakube/monitoredentities/reconciler.go

```diff
@@ -0,0 +1,67 @@
+package monitoredentities
+
+import (
+	"context"
+	"errors"
+
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3/dynakube"
+	"github.com/Dynatrace/dynatrace-operator/pkg/clients/dynatrace"
+	"github.com/Dynatrace/dynatrace-operator/pkg/controllers"
+)
+
+type ReconcilerBuilder func(
+	dtClient dynatrace.Client,
+	dk *dynakube.DynaKube,
+) controllers.Reconciler
+
+func NewReconciler( //nolint
+	dtClient dynatrace.Client,
+	dk *dynakube.DynaKube,
+	kubeSystemUUID string,
```

Comment:
not needed

Reply:
done in https://github.com/Dynatrace/dynatrace-operator/pull/3447/commits/8fcfcc3ee0a69d518855231cd6151675b1e180bb